plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -72,28 +72,33 @@ def settings(self, hostSession):
 
     def initialize(self, managerSettings, hostSession):
         bal.validate_settings(managerSettings)
-        self.__settings.update(managerSettings)
 
-        self.__library = {}
+        # Settings updates can be partial, so make sure we keep any
+        # existing path.
+        existing_library_path = self.__settings.get("library_path")
+        library_path = managerSettings.get("library_path", existing_library_path)
 
-        if self.__settings.get("library_path") is None:
+        if not library_path:
             hostSession.logger().log(
                 hostSession.logger().Severity.kDebug,
-                f"'library_path' not in settings, checking {self.__lib_path_envvar_name}",
+                f"'library_path' not in settings or empty, checking {self.__lib_path_envvar_name}",
```

Comment:
[Consider][Minor]
`not in settings or empty` is slightly ambiguous, as in, NOT (in setting || empty) or NOT (in settings) || empty,  `not in settings or is empty` clarifies to my eye.

Reply:
Good call, fixed [here](https://github.com/OpenAssetIO/OpenAssetIO-Manager-BAL/pull/27/commits/9ce7f5cfd83bfa4bbbdee84148d2853e926b6263).